Makefile

```diff
@@ -85,4 +85,12 @@ bdddeps:
 	@go get -u github.com/onsi/ginkgo
 	@go get -u github.com/onsi/gomega 
 	kubectl create -f https://raw.githubusercontent.com/litmuschaos/chaos-operator/master/deploy/chaos_crds.yaml
-	kubectl create ns litmus
\ No newline at end of file
+	kubectl create ns litmus
+
+.PHONY: trivy-security-check
+trivy-security-check:
+	@echo "------------------"
+	@echo "--> Trivy Security Check"
+	@echo "------------------"
+	trivy --exit-code 0 --severity HIGH --no-progress litmuschaos/ansible-runner:ci
```

Comment:
Why are we using the ansible-runner image here?

Reply:
Changed